src/cljs/chronograph_web/http.cljs

```diff
@@ -1,6 +1,15 @@
 (ns chronograph-web.http
   (:require [ajax.core :as ajax]))
 
+(defn GET
+  [uri m]
```

Comment:
Let's avoid single character argument names?

Reply:
OK will alter it. I followed the `post` fn that neena wrote, for consistency. I'm usually fine with well-understood arg names like `f / s / x / xs / m / [k v]`.

Reply:
Revised argument names